pkg/tinyusb/doc.txt

```diff
@@ -0,0 +1,79 @@
+/**
+ * @defgroup pkg_tinyusb TinyUSB package
+ * @ingroup  pkg
+ * @brief    Provides the tinyUSB stack as package
+ * @author   Gunar Schorcht <gunar@schorcht.net>
+ * @see https://github.com/hathach/tinyusb
+ *
+ * # TinyUSB
+ *
+ * tinyUSB is an open-source cross-platform USB Host/Device stack for
+ * embedded systems.
+ *
+ * # Usage
+ *
+ * Add the following entries to your application makefile:
+ * - Enable tinyUSB package
+ *   ```makefile
+ *   USEPKG += tinyusb
+ *   ```
+ * - Select whether to use the tinyUSB device stack or the tinyUSB host stack or both:
+ *   ```makefile
+ *   USEMODULE += tinyusb_device
+ *   ```
+ * - Select the device/host classes to be used, for example:
+ *   ```makefile
+ *   USEMODULE += tinyusb_class_cdc tinyusb_class_msc
+ *   ```
+ *
+ * Add `tinyusb_setup()` to your main function to initialize the tinyUSB stack
+ * including used peripherals and to start the tinyUSB thread.
+ * ```c
+ * int main(void)
+ * {
+ *     ...
+ *     // initialize the tinyUSB stack including used peripherals and
+ *     // start the tinyUSB thread
+ *     tinyusb_setup();
```

Comment:
Why is this not auto-init'ed?

Reply:
Do you mind if we add this in a followup PR ?
This PR is already in good shape.

Reply:
> Why is this not auto-init'ed?

Good question. If it is possible to create a thread during the auto-init and the scheduler is already running, it should be possible.

Reply:
I have a local commit that implements auto-initialization. It only needed a few changes:
```
 pkg/tinyusb/Makefile.dep                     |  2 ++
 pkg/tinyusb/contrib/tinyusb.c                | 13 +++++++++++++
 pkg/tinyusb/doc.txt                          |  6 ++++--
 sys/auto_init/include/auto_init_priorities.h |  6 ++++++
 tests/pkg_tinyusb_cdc_msc/main.c             |  4 ----
 5 files changed, 25 insertions(+), 6 deletions(-)
```
Should I provide it or should we add this in a followup PR?

Reply:
See commit 154152d55497a0cdfcf1ff1aacbffc5a5e4c0b47.